server/controllers/BookController.js

```diff
@@ -17,25 +17,26 @@ class BookController {
    */
   static async addBook(request, response) {
     const {
-      title, description, isbn, price, yearPublished, stock, authorName
-    } = request.body;
+      body: {
+        title, description, isbn, price, yearPublished, authorName
+      }
+    } = request;
```

Comment:
What of if you have several authors to a book, how would that be tackled

Reply:
I am not the one that implemented this feature. Books asked me to help him change somethings in my PR. Please speak to books.

Reply:
kindly revert back so biola can create a bug fix for this